lib/diagnostic.rb

```diff
@@ -22,13 +22,17 @@ def example
 # your answer here
 
 #
-# Question 2
+# Behavior Driven Developement is writing tests that check for how the given code should behave instead of whether is works or not.
 #
 # Create a request spec for our Examples API that ensures 'GET /examples'
 # responds successfully and lists all examples.
 
-RSpec.describe 'Examples API' do
-  # your test(s) here
+RSpec.describe 'GET /examples API' do
+  it 'lists all examples' do
+    get '/examples'
+
+    expect(response).to be_success
```

Comment:
Want to make sure we also check that the examples coming back from the successful response are what we want with one or both of:

```
examples_response = JSON.parse(response.body)
expect(examples_response.length).to eq(examples.count)
expect(examples_response['examples'].first['title']).to eq(example[:title])
```